src/web/components/PluginInjectionSite/module.jsx

```diff
@@ -47,15 +56,29 @@ export default class InjectedModuleView extends React.Component {
   }
 
   setViewInState(moduleName, viewName, isLite) {
-    const fullModuleName = moduleName.startsWith('botpress-') ? moduleName : 'botpress-' + moduleName
+    const lookupNames =
+      moduleName.startsWith('@botpress/') || moduleName.startsWith('botpress-')
+        ? [moduleName]
+        : ['@botpress/' + moduleName, 'botpress-' + moduleName]
+
+    const viewResolve = name => {
+      const prop = isLite ? 'default' : viewName
+      return window.botpress && window.botpress[name] && window.botpress[name][prop]
+    }
 
-    const module = isLite
-      ? window.botpress && window.botpress[fullModuleName].default
-      : window.botpress && window.botpress[fullModuleName] && window.botpress[fullModuleName][viewName]
+    let module = null
+
+    for (const name of lookupNames) {
+      const m = viewResolve(name)
+      if (m) {
+        module = m
+        break
+      }
+    }
```

Comment:
@slvnperron , can it be done via `Array#find`? Like `const module = lookupNames.find(viewResolve)`.

Reply:
@epaminond genius! Didn't know array.find was introduced in ES6